server/server.go

```diff
@@ -458,7 +459,7 @@ func (server HTTPServer) readAggregatorRuleForClusterID(
 	return aggregatorResponse.Report, true
 }
 
-func (server HTTPServer) fetchRuleContent(rule types.RuleOnReport) (proxy_types.RuleWithContentResponse, bool) {
+func (server HTTPServer) fetchRuleContent(rule types.RuleOnReport, OSDEligible bool) (proxy_types.RuleWithContentResponse, bool, bool) {
```

Comment:
I think it's better to document the return options since it's not very obvious what they are without reading the code

Reply:
Yes, sure

Reply:
dunno if it is not better to return error value with proper information about the failure

Reply:
But information about failure is handled before in inner functions.

Reply:
ok agree. Could you then please name the return values? TYVM

Reply:
@tisnik I already left a comment, is there enough, check it please?

Reply:
ok not a problem